crates/astria-composer/src/searcher/collector.rs

```diff
@@ -75,8 +76,14 @@ impl Collector {
         new_bundles: Sender<SequenceAction>,
     ) -> Self {
         let (status, _) = watch::channel(Status::new());
+        let rollup_id = RollupId::from_unhashed_bytes(&chain_name);
+        info!(
```

Comment:
Commenting here because github doesn't permit commenting far away from diffs:

I think `rollup_id` should be injected into `Collector::run_until_stopped`, and a message generated there, like "running collector` or something like that.